.travis.yml

```diff
@@ -9,12 +12,18 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=root TEST_TYPE=functional
+  - TEST_DIR=utils
+
 install:
   - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
   # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
   - ansible-playbook --syntax-check playbooks/byo/config.yml
-  - tox
-  - cd utils && tox
+  - if [[ "${TEST_DIR}" == "root" ]]; then tox; fi
+  - if [[ "${TEST_DIR}" == "utils" ]]; then cd utils && tox; fi
```

Comment:
Isn't this getting too confusing? I liked better to run `make ci` locally. Using `env` might be good, but at the same time this is making it harder for people to understand what is running in Travis and being able to reproduce it locally before commit/push.

The `utils` feels like it wants to be it's own repo... can't we run the tests for `aos-installer` (utils) from the repository root?